source/exe/stripped_main_base.cc

```diff
@@ -70,8 +70,6 @@ StrippedMainBase::StrippedMainBase(const Server::Options& options, Event::TimeSy
 
     tls_ = std::make_unique<ThreadLocal::InstanceImpl>();
     Thread::BasicLockable& log_lock = restarter_->logLock();
-    Thread::BasicLockable& access_log_lock = restarter_->accessLogLock();
```

Comment:
We removed the `access_log_lock` here and reuse `log_lock` in the position where originally need `access_log_lock`. Why?